docs/source/guides/logical_types_and_semantic_tags.ipynb

```diff
@@ -0,0 +1,693 @@
+    "\n",
+    "The `parent_type` column is indicitive of how type inference should rank inferred types since it's possible for data to be inferred as more than one `LogicalType`. For example: A column of all integers with no null values could be inferred as either `Integer` or `IntegerNullable`. `IntegerNullable` being the parent of `Integer` indicates that we know more about the contents of a column if it's able to be an `Integer` column (namely, that there are no null values), and Woodwork should infer a column as the `LogicalType` for which we know the most information.\n",
+    "\n",
+    "#### Base LogicalType Class\n",
+    "\n",
+    "All logical types used by Woodwork are subclassed off of the base `LogicalType` class, and since the following behaviors all exist on the `LogicalType` class, all logical types have the following behavior:\n",
+    "\n",
+    "- All logical types define a `dtype` that will get used for any column with that logical type - this is how the physical type for a column gets determined\n",
+    "- All logical types perform some basic transformation into the expected physical type (`dtype`) - this is how Woodwork LogicalTypes act as a form of data-transformers. Depending on the requirements of a LogicalType, a LogicalType can transform input data into an expected format. \n",
+    "\n",
+    "    ```python\n",
+    "    class LogicalType(object, metaclass=LogicalTypeMetaClass):\n",
+    "        \"\"\"Base class for all other Logical Types\"\"\"\n",
+    "        type_string = ClassNameDescriptor()\n",
+    "        primary_dtype = 'string'\n",
+    "        backup_dtype = None\n",
+    "        standard_tags = set()\n",
+    "    ```\n",
+    "\n",
+    "#### Default Logical Type\n",
+    "\n",
+    "##### Unknown\n",
+    "When Woodwork's type inference does not return any LogicalTypes for a column, Woodwork will set the column's logical type as the default LogicalType, `Unknown`. A logical type being inferred as `Unknown` may be a good indicator that a more specific logical type can be chosen and set by the user.\n",
+    "- **physical type**: `string`\n",
+    "\n",
+    "Below is an example of a column for which no logical type is inferred, resulting in a Series with `Unknown` logical type. Looking at the contents of the Series, though, we can see that it contains country codes, so we set the logical type to `CountryCode`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bottom-alliance",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "series = pd.Series([\"AUS\", \"USA\", \"UKR\"])\n",
```

Comment:
Need to update these values since we changed the definition for the `CountryCode` logical type.